dotnetv3/KMS/DisableKeyExample/DisableKey.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+/// <summary>
+/// Disable an Amazon Key Management Service key and then retrieve the key's
+/// status to show that it has been disabled. This example was created using
+/// the AWS SDK for .NET and .NET Core 5.0.
+/// </summary>
+namespace DisableKeyExample
+{
+    // snippet-start:[KMS.dotnetv3.DisableKeyExample]
+    using System;
+    using System.Threading.Tasks;
+    using Amazon.KeyManagementService;
+    using Amazon.KeyManagementService.Model;
+
+    public class DisableKey
+    {
+        public static async Task Main()
+        {
+            var client = new AmazonKeyManagementServiceClient();
+
+            // The identifier of the KMS key to disable. You can use the
+            // key ID or the Amazon Resource Name (ARN) of the KMS key.
+            var keyId = "1234abcd-12ab-34cd-56ef-1234567890ab";
+
+            var request = new DisableKeyRequest
+            {
+                KeyId = keyId,
+            };
+
+            var response = await client.DisableKeyAsync(request);
+
+            if (response.HttpStatusCode == System.Net.HttpStatusCode.OK)
+            {
+                // Retrieve information about the key to show that it has now
+                // been disabled.
+                var describeResponse = await client.DescribeKeyAsync(new DescribeKeyRequest
+                {
+                    KeyId = keyId,
+                });
+                Console.WriteLine($"{describeResponse.KeyMetadata.KeyId} - state: {describeResponse.KeyMetadata.KeyState}");
+            }
+        }
+    }
+
```

Comment:
Remove this blank line.

Reply:
Again, it's there because of the linter.